spec/controllers/assignments_controller_spec.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-require 'rails_helper'
```

Comment:
I know removing `rails_helper` has no functional difference for us but, the thing that bothers me about this is rails thinks we should have both. Plus from what I've read, mainly [here](https://stackoverflow.com/questions/24145329/how-is-spec-rails-helper-rb-different-from-spec-spec-helper-rb-do-i-need-it), the `rails_helper` should be required for any test that depends on rails and `spec_helper` should be required for those that don't.

Reply:
I read that too. However,

> spec_helper.rb is for specs which don't depend on Rails (such as specs for classes in the lib directory)

Are you writing tests for classes in the lib directory and running them in isolation much?

> rails thinks we should have both

Well, a lot of people write rails, and yeah they have their reasons for doing this, the reason you stated being the only functional one I could discern, but I don't buy it enough. For this application, which has tests that are pretty much all about Rails, there would be no benefit. For larger projects with a lot of non-rails tests, sure. Rails was built to be flexible, and having two helper files for specs allows that. But, that flexibility also allows us to remove it.

We can get another opinion, but for now I'm gonna say if you want to have both, figure out which should go in either of them and go nuts... because I'm done with this PR. :)